pkg/theiactl/commands/check.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	sparkv1 "antrea.io/theia/third_party/sparkoperator/v1beta2"
+)
+
+// checkCmd represents the check command
+var checkCmd = &cobra.Command{
+	Use:   "check",
+	Short: "Check the status of a policy recommendation Spark job",
+	Long: `Check the current status of a policy recommendation Spark job by ID.
+It will return the status of this Spark application like SUBMITTED, RUNNING, COMPLETED, or FAILED.`,
```

Comment:
Not sure what does "Spark application" mean, is it different with "Spark job"?

Reply:
Spark application is the name of CRD defined by Spark Operator. Sorry to be confusing here, I will rephrase these comments.